StripeCore/StripeCore/Source/API Bindings/STPAPIClient.swift

```diff
@@ -87,6 +87,10 @@ import UIKit
         return publishableKey?.hasPrefix("uk_") ?? false
     }
 
+    /// Determines the `Stripe-Livemode` header value when the publishable key is a user key
+    /// :nodoc:
+    @_spi(StripeAppsOnly) public var userKeyLiveMode = true
```

Comment:
I thought about calling this `DashboardOnly` inline with the `LinkOnly` notation, but I think this might be used by more than just dashboard... I'm not totally sure TBH

Reply:
Seems fine. I think you could also call this @_spi(STP) too.

Reply:
I considered using `STP` but I think that anything marked as `STP` is implicitly internal and could change without warning. Whereas marking it as `StripeAppsOnly` or `DashboardOnly` implies that someone else is using this and to be wary of making breaking changes.

I am leaning towards `DashboardOnly` now since I think this is consistent with internal naming for API fields that use the `uk_` key.